IBPSA/Fluid/HeatExchangers/BaseClasses/PartialEffectivenessNTU.mo

```diff
@@ -20,7 +20,7 @@ model PartialEffectivenessNTU
                 Dialog(group="Nominal thermal performance"));
 
   parameter Modelica.SIunits.HeatFlowRate Q_flow_nominal(fixed=use_Q_flow_nominal)
-    "Nominal heat transfer"
+    "Nominal heat transfer from port 1 to port 2 (positive if port 2 is heating up)"
```

Comment:
I suggest `Nominal heat flow rate (positive for heat transfer from 1 to 2)`.

Reply:
Done.